tests/unit/test-module-for-component-test.js

```diff
@@ -760,7 +763,7 @@ moduleForComponent('Component Integration Tests: willDestoryElement', {
       'component:my-component': Component.extend({
         willDestroyElement() {
           var stateIndicatesInDOM = this._state === 'inDOM';
-          var actuallyInDOM = $.contains(document, this.$()[0]);
+          var actuallyInDOM = document.contains(this._element);
 
           QUnit.config.current.assert.ok(
```

Comment:
Should probably break this up into two assertions so you can get more granular feedback when it fails.

Reply:
Ya, great point!